pykokkos/lib/ufunc_workunits.py

```diff
@@ -1,6 +1,513 @@
 import pykokkos as pk
 
 
+@pk.workunit
+def less_impl_5d_int8(tid: int,
+                              view1: pk.View5D[pk.int8],
+                              view2: pk.View5D[pk.int8],
+                              out: pk.View5D[pk.uint8]):
```

Comment:
column-alignment could be cleaned up--feel free to help me with that... wasn't a high priority of course